server/db/index.js

```diff
@@ -0,0 +1,19 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+import connectionString from './dbConfig';
+
+dotenv.config();
+
+const env = process.env.NODE_ENV;
+console.log(env);
+let connection;
+
+if (env === 'development') {
+  connection = connectionString.development;
+} else if (env === 'production') {
+  connection = connectionString.production;
+}
+
```

Comment:
you can do this to include the test environment:

if (env === 'production') {
  connection = connectionString.production;
} else if (env === 'test') {
  connection = connectionString.test;
} else {
  connection = connectionString.development;
}